apps/api/docs/block-diagram.md

```diff
@@ -2,21 +2,21 @@
 
 ## Introduction
 
-This document provides a concise block diagram illustrating the architecture of our software application, Osmo-Notify. The diagram outlines the key components and their interactions to help you better understand how the application functions.
+This document provides a concise block diagram illustrating the architecture of our software application, OsmoX. The diagram outlines the key components and their interactions to help you better understand how the application functions.
 
 ## Block Diagram Overview
 
 The block diagram represents the software application's architecture in a visual format, making it easier to grasp its structure and functionality.
 
 ## Block Diagram
 
-![Block Diagram](./assets/osmo-notify-block-diagram.png)
+![Block Diagram](./assets/osmoX-block-diagram.png)
```

Comment:
make it  "OsmoX-block-diagram.png."

Reply:
Addressed in https://github.com/OsmosysSoftware/osmo-x/pull/152/commits/ffca8627cb4ae9663e16c6d1288440193d1ada1b